doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_features.rst

```diff
@@ -0,0 +1,176 @@
+However, we also apply optimizations to each instance of each processor, so that CPUs are tailored to their range of tasks.
+This way, you can freely select the CPU that is best suited to the various workloads that make up an application.
+Each CPU runs its own instance of the operating system.
+There is no centralized scheduler that assigns threads to various cores.
+This means that you have a full control over which CPU is used for a given task.
+
+System services
+===============
+
+One key feature of the nRF54H20 is that a lot of functionality is provided as services, implemented by Nordic-provided firmware.
+This means that users access functionality through a software API rather than through a set of registers.
+Using a firmware layer means that functionality can be more advanced and handled at a higher level of abstraction, which increases ease-of-use.
+It is also more flexible and updateable than with the purely hardware approach.
+
+Power Management Service
+========================
+
+Rather than manually setting a power mode through a register or calling a sleep instruction, power management is performed through a service where the application describes the functionality it needs and how much latency it can accept.
+Based on this information, the Power Management Service automatically handles the details, deciding how registers will be retained, which parts of the device are put into what mode, and what clock signals are running.
+
+Security model, assets, threats, protections
+============================================
+
+The nRF54H20 SoC security architecture is based on the conclusions from extensive threat modelling.
+nRF54H20 is designed to protect against logical attacks and firmware-oriented attacks, as well as physical security at the PCB or package level (for example, against voltage glitching and EMI attacks).
+However, robustness against physical die probing or alteration of the die itself is not provided by the design.
+Security in software is implemented as a set of services, provided by the Secure Domain.
+These services manage Secure Boot, cryptographic operations, key management, and firmware updates.
+The main application and other parts of the system access security services through an API.
+This API is translated internally to messages sent to the Secure Domain through a mailbox-based interface.
+The security services handle these requests, utilizing hardware accelerators and other resources, and as security is centralized in the Secure Domain, keys can be kept inside this physical area and remain well-protected against attacks or breaches due to firmware exploits.
+nRF54H20 also includes tamper sensors that can detect an attack in progress and take appropriate action.
+The cryptographic accelerators have been hardened against side-channel attacks.
+
+CPUs overview and features
```

Comment:
Maybe this can be moved further up in the doc, perhaps even as the first subheading